sql/changes/LOADORDER

```diff
@@ -54,11 +54,10 @@
 1.5/fixed-assets-not-null-constraints.sql
 1.5/template_menu-v2.sql
 1.5/abstract_tables.sql
-#tag: migration-target
```

Comment:
The text block below belongs to the `migration-target` tag. You're moving the tag, but editing the text of the textblock without moving it with the tag?

Also, since this change implements SL3.2 migration, shouldn't the "old" tag stay where it is (to support the old migrations)?